objects/objects.py

```diff
@@ -226,7 +226,10 @@ def get_defaults(self, name):
     'syun_',
     {'folder': path_to_SVG / 'syun'}
 )
-
+svg_defaults.add_default(
+    'door',
+    {'folder': path_to_SVG}
+)
```

Comment:
This is the default, no need to specify

Reply:
What you mean? specify what?

Reply:
You don't need to add those 4 lines, it's the default already.